src/game/gui/progressbar.c

```diff
@@ -52,12 +53,16 @@ typedef struct {
     int refresh;
 } progressbar;
 
-void progressbar_set_progress(component *c, int percentage) {
+void progressbar_set_progress(component *c, int percentage, bool animate) {
```

Comment:
Code docs on what the arguments do ? Took a while to read through it, so quick comments might help here.